### ISSUE TITLE
Python 3.13 Support

### ISSUE THREAD

**Issue by ShikiOkasaka** [polm/fugashi#96]
It would be nice if we could install fugashi on Fedora 41 from wheels with python 3.13.0.

**Comment by polm** [comment]
Thanks for pointing that out! I should be able to get this up this week.

**Comment by polm** [comment]
This should be available with `pip install fugashi==1.3.3.dev1` right now (or in a few minutes). I'll do some checks and make a proper release, but if you could confirm it works that would help.

**Comment by ShikiOkasaka** [comment]
It works fine on my Fedora 41 machine:
```
$ pip install fugashi==1.3.3.dev1
$ python
Python 3.13.0 (main, Oct  8 2024, 00:00:00) [GCC 14.2.1 20240912 (Red Hat 14.2.1-3)] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from fugashi import Tagger
```

**Comment by polm** [comment]
Thanks for the confirmation! This should be all fixed and will shortly be available as v1.3.3.
